README.md

```diff
@@ -72,18 +72,17 @@ you declared in the igress.yaml in your `notes-app` repository, e.g. `http://<gi
 ## Step 8: Monitoring
 Monitoring of important metrics and logs is essential to successfully implement DevOps. Developers and other stakeholders must be able to observe the metrics in specialized dashboards, and receive notifications when certain metrics exceeds a defined threshold or if a failure switch has been triggered. Good
 dashboards can provide developers with important insight into parameters such as performance, usage patterns, resource capacity, stability during peak hours
-etc. There are many tools that one can use to set up monitoring with, and we will cover the basics of one such toolset, namely Grafana and Prometheus. 
+etc. There are many tools that one can use to set up monitoring with, and we will cover the basics of one such toolset, namely Grafana and Prometheus.
```

Comment:
Sorry bout the whitespace changes, but my editor is set to trim trailing space :sweat_smile: